dist/plugin.js

```diff
@@ -0,0 +1,5 @@
+const rules = require("./rules");
```

Comment:
whats the need for this?

Reply:
for this plugins entrypoint

Reply:
this is the version we came to after discussing with the eslint discord. The idea is to have a base plugin which has the rules and any other objects in the future we may want to ship with the package. We can the reference this with the configs we make, avoiding the circular reference issue. The package then exports the configs and base plugin